Dockerfile

```diff
@@ -18,4 +31,4 @@ RUN groupadd --system --gid=9999 mcad && \
 
 RUN chown -R mcad:mcad /usr/local/bin
 
-USER mcad
+#USER mcad
```

Comment:
Can we jsut delete it altogether? Related, why are we creating an mcad user specifically? Couldn't we just do something like `USER 65532:65532`

Reply:
Shouldn't we also have an entrypoint defined?

Reply:
@anishasthana 
Commenting out that line was a mistake from my part. I have reverted that changed.

I am not sure why the previous version was using the user mead, but I see no harm in it.  Is there any benefit to using `USER 65532:65532` ?

In regards to the entry point issue, the helm chart does define the command line to use when starting the MCAD controller, so to run it there's no need for an entry point.

@asm582, @jbusche and @dmatch01 should we add an entry point now ?

Reply:
@z103cb agree helm command line options are used to start the MCAD controller. entrypoint addition is not needed for this release.